serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc

```diff
@@ -273,25 +273,45 @@ When using the Kubernetes service discovery feature, you need to balance if your
 
 If the URI pattern is not found in the application properties, then discovery is not triggered. However, the scanning is performed anyway. Therefore, a short time is required to go through the startup scan.
 
+=== Available service discovery implementations
+
+Currently, there are two implementations available.
+
 You can enable the Kubernetes service discovery by adding a service discovery implementation to the application's dependencies as shown in the following Maven dependencies:
 
+==== Fabric8 Kubernetes Service Catalog
```

Comment:
```suggestion
==== Fabric8 Kubernetes service catalog
```

Reply:
That's the name of the module. So, I think uppercase is fine.